docs/reference/metrics.md

```diff
@@ -0,0 +1,115 @@
+# Metrics
+
+TODO: intro text.
+
+TODO: move - Because ksqlDB persistent queries directly compile into {{
+site.kstreams }} topologies, many ksqlDB metrics correspond to [Kafka
+Streams metrics](https://docs.confluent.io/current/streams/monitoring.html).
```

Comment:
I realize this part is within a TODO, but I am curious if this means that ksqlDB server directly provides the `kafka.streams:*` MBeans listed on the linked page. For instance, should I expect thread and task metrics to be provided directly by ksqlDB's JMX endpoint, or would I have to gather them from JMX on the broker?

Reply:
I'm going to dig in and find out. :)

Reply:
> I am curious if this means that ksqlDB server directly provides the kafka.streams:*

It does.